docs/controls/wpf-winforms/WebView.md

```diff
@@ -35,29 +35,25 @@ Unless specified otherwise in this article, the documentation for the [WebViewCo
 
 When compared to the UWP **WebView** control, the current release of the WPF and Windows Forms **WebView** control has some limitations. For the complete list of these limitations, see [Known Issues of the WebView control for Windows Forms and WPF applications](WebView-known-issues.md).
 
-## Add the WebView control to the Visual Studio Toolbox
+## Add the WebView control to the Visual Studio Toolbox for Windows Forms applications
 
-First, open the Visual Studio **Toolbox**, then right-click anywhere in the toolbox, and select the **Choose Items** option.
+1. Install the [Microsoft.Toolkit.Forms.UI.Controls.WebView](https://www.nuget.org/packages/Microsoft.Toolkit.Forms.UI.Controls.WebView) NuGet package.
 
-<!-- Remove these steps before going public. This is for early adopters only -->
+2. Open the Visual Studio **Toolbox**, right-click anywhere in the toolbox, and select the **Choose Items** option.
 
-### For Windows Forms applications
+2. In the **.NET Framework Components** tab of the **Choose Toolbox Items** dialog box, click **Browse** to locate the **Microsoft.Toolkit.Forms.UI.Controls.WebView.dll** in your [NuGet package folder](https://docs.microsoft.com/nuget/consume-packages/managing-the-global-packages-and-cache-folders).
```

Comment:
These steps no longer required with Visual Studio 15.8

Reply:
Thanks for the feedback. I split up the Windows Forms steps for VS 15.8 (just install the package and the control shows up in the toolbox) and earlier versions of VS (install the package and manually add the assembly to the toolbox).